src/Bundle/Resources/config/services/expression_language.xml

```diff
@@ -13,28 +13,36 @@
 
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <services>
+        <service id="sylius.metadata.expression_language" class="Symfony\Component\ExpressionLanguage\ExpressionLanguage" />
```

Comment:
Should it be like so, since rest of the services are type prefixes, not "domain" prefixes, like `sylius.resource_factory`, `sylius.repository`, `sylius.routing`.

```suggestion
        <service id="sylius.expression_language.metadata" class="Symfony\Component\ExpressionLanguage\ExpressionLanguage" />
```

Plus it looks weird in combination with other new services, like

```
sylius.metadata.expression_language
sylius.expression_language.variables_collection.metadata
sylius.expression_language.vars_resolver.metadata
sylius.expression_language.argument_parser.metadata
```